### ISSUE TITLE
MacOS Dependency file doesn't include Python37 Lib

### ISSUE THREAD

**Issue by camerontarget14** [PrismPipeline/Prism#39]
Hi,

First of all thank you so much for this program! I'm so excited to get started, this truly is a gift to the VFX industry.

My studio is thinking of adopting this software, but we need it to be cross platform compatible to run on the few Macs we have. 

Do you have any plans to update the software so it's compatible with the latest MacOS?

Thank you thank you thank you!

**Comment by RichardFrangenberg** [comment]
We are currently focusing on the Windows version of Prism 2. If there is enough interest for a Mac or Linux version it will be considered as well, but it's currently not planned for the near future.

**Comment by camerontarget14** [comment]
Thanks! Does that include Windows 11? 

Also, I got it to work! but I got this error on my Mac, wondering if you had a quick fix.


<img width="1096" alt="Screenshot 2023-05-04 at 8 30 25 AM" src="https://user-images.githubusercontent.com/127965975/236204476-76831f98-fe39-4e7f-ae34-ba0e8955abb6.png">


Thanks again for your time.

**Comment by RichardFrangenberg** [comment]
Yes w10 and w11 will be both fully supported. I already switched to w11 on my development PC.

Your warning looks like Prism send a request to a url, which doesn't exist anymore. Doublecheck that your Shotgrid url is correct.
You can also try to update the shotgun api module in your Prism installation here: https://github.com/RichardFrangenberg/Prism/tree/development/Prism/Plugins/ProjectManagers/Shotgun/external_modules/shotgun_api3
With the latest version from here: https://github.com/shotgunsoftware/python-api
